src/server/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -63,12 +82,27 @@ namespace ts.server.typingsInstaller {
                 this.log.writeLine(`Process id: ${process.pid}`);
             }
             this.npmPath = getNPMLocation(process.argv[0]);
-            this.exec = require("child_process").exec;
-            this.httpGet = require("http").get;
+            let execSync: ExecSync;
+            ({ exec: this.exec, execSync } = require("child_process"));
+
+            this.ensurePackageDirectoryExists(globalTypingsCacheLocation);
+
+            try {
+                if (this.log.isEnabled()) {
+                    this.log.writeLine(`Updating ${TypesRegistryPackageName} npm package...`);

```

Comment:
Actually - scratch that. I just tested, and at least with the npm I have, `npm install` did actually update it (always - even if already up to date), and `npm update` did nothing. Perhaps because I'm not specifying it as a dependency with a version in package.json (I didn't --save or --save-dev so it's not listed, just as you haven't below). So this seems fine as is.